legend-engine-xts-dataquality/legend-engine-xt-dataquality-pure-test/src/main/resources/core_dataquality_test/dataquality_test_utils.pure

```diff
@@ -63,7 +85,15 @@ function meta::external::dataquality::tests::assertLambdaEquals(expected:Functio
                                   ]
                             )
             );
+  assertEquals($expected->printFunctionDefinition($config, ''), $actual->printFunctionDefinition($config, ''));
```

Comment:
can we reuse the existing assertLambdaAndJSONEquals/assertLambdaEquals function?

Reply:
Used existing lambda equality function.